group_vars/ssh.yml

```diff
@@ -1,73 +1,112 @@
 ---
 
+# I need someone to complete this list bellow because Im probably missing most of them
+ssh_roles:
+  - name: admin
+    hosts: all
+    users:
+      - centos
+      - stats
+      - galaxy
+      - root  # can be omitted for now, centos can `sudo su`
+
+  - name: stats
+    hosts:
+      - sn09
+      - sn06
+    users:
+      - stats
+
+  - name: galaxy
+    hosts:
+      - sn09
+      - sn06
+    users:
+      - galaxy
+
 ssh_users:
   - user: galaxy
     key:
```

Comment:
I'd suggest using `keys:` instead of `key:` so it's clear that the role expects a list of keys rather than just one.